infra-templates/portworx/1/rancher-compose.yml

```diff
@@ -0,0 +1,56 @@
+.catalog:
+  name: "Portworx"
+  version: "1.2.0"
```

Comment:
version field should match version field at infra-templates/portworx/config.yml
```
-  version: "1.2.0"
+  version: "1.2.11.1"
```

Reply:
I will update the same. thanks